examples/rstudio/test.bats

```diff
@@ -0,0 +1,32 @@
+true
+# shellcheck disable=SC2034
+CH_TEST_TAG=$ch_test_tag
+
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope full
+    prerequisites_ok "rstudio"
+}
+
+@test "${ch_tag}/test rstudio-server" {
+    # Start up on port 8991
+    run ch-run "$ch_img" -- start_rstudio 8991 &
+    rstudio_server=$!
+    run ch-run "$ch_img" -- python3 test_rstudio.py
+    echo $output
+    [[ $status -eq 0 ]]
+    [[ $output = *'Good'* ]] 
+    kill -9 $rstudio_server
```

Comment:
1. Don't `kill -9` unless plain `kill` doesn't work, because it won't let the process clean up.
2. How will the next test work if the Rstudio server has been stopped?

Reply:
1. Ah! Good point. 
2. The next test is independent from the rstudio server. It tests the R installation on the system that Rstudio uses. I explain it further [here](https://github.com/hpc/charliecloud/pull/1188/files/3f26830644f93a6aa59284818d2f1ba18ef481d2#r721523859).